openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -125,3 +121,7 @@
 <%static:require_module_async module_name="js/dateutil_factory" class_name="DateUtilFactory">
     DateUtilFactory.transform('.localized-datetime');
 </%static:require_module_async>
+
+<%static:webpack entry="CourseOutline">
+    new CourseOutline();
```

Comment:
IMO this should pass ``'.block-tree'`` like the old factory, so that it is possible to render the outline in different contexts.